.github/workflows/tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         #    python-version: 3.8
 
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2
```

Comment:
my guess is that master is always up-to-date whereas V2 might get out-dated when V3 comes out

Reply:
yeahh, i was just trying to see if it makes any difference. Reverting back to master

Reply:
master is the latest development and might contain bugs. So normally its preferable to use a released and numbered version like v2.

Reply:
ok, but that means from time to time we'll need to check if there is a V3 out? we can change to V2 for checkout and the other action in all of the workflows